ext/posix/tests/posix_setegid_basic.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Test function posix_setegid() by calling it with its expected arguments
+--SKIPIF--
+<?php 
+    if(!extension_loaded("posix")) die("skip - POSIX extension not loaded"); 
+?>
+--FILE--
+<?php
+var_dump(posix_setegid(posix_getegid()));
```

Comment:
could you please add something like this
```
$oldId = posix_getegid();
posix_setegid(42);
var_dump(posix_getegid() === 42);
posix_setegid($oldId);
var_dump(posix_getegid() === $oldId);
```

at least we will make sure that it really works as expected